### ISSUE TITLE
Unable to parse cloudformation template

### ISSUE THREAD

**Issue by Tasemu** [localstack/localstack#1572]
Hi, i have a cloudformation template that is not working with localstack. It works fine until I add this block, after much commenting out of lines, i have worked out that is creates the stack fine as long as I do not have the lines containing !FindInMap... This looks totally fine to be but localstack breaks and throws an unable to parse error.

It is also worth noting that I am using the same block of code under LogGroupName in another resource and it is working totally fine. It just seems to be for the Log Group resource.

```
# SEEMS AS THOUGH FINDINMAP IS BREAKING THIS
  HeartbeatHandlerLogGroup:
    Type: AWS::Logs::LogGroup
    DependsOn:
      - FooHandler
    Properties:
      RetentionInDays: !Ref FooRetention
      LogGroupName:
        Fn::Join:
          - '_'
          - - '/aws/lambda/AWS_DUB_LAM'
            - !FindInMap [ AccountInfo, !Ref "AWS::AccountId", ID ]
            - !If [ FeatureBranch, !Ref "gitBranch", !Ref "AWS::NoValue" ]
            - 'MessageFooHandler'
            - !FindInMap [ AccountInfo, !Ref "AWS::AccountId", ENV ]

# HOWEVER THIS RESOURCE IS WORKING WITH THE SAME LOGGROUPNAME
  LogGroupSubscriptionHeartbeatService:
    Type: AWS::Logs::SubscriptionFilter
    DependsOn:
      - FooLogGroup
    Properties:
      DestinationArn: !Ref MessageServicelogFooArn
      FilterPattern: ''
      LogGroupName:
        Fn::Join:
          - '_'
          - - '/aws/lambda/AWS_DUB_LAM'
            - !FindInMap [ AccountInfo, !Ref "AWS::AccountId", ID ]
            - !If [ FeatureBranch, !Ref "gitBranch", !Ref "AWS::NoValue" ]
            - 'MessageFooHeartbeatHandler'
            - !FindInMap [ AccountInfo, !Ref "AWS::AccountId", ENV ]
```

**Comment by whummer** [comment]
Thanks for reporting @Tasemu . Can you please provide some more details about the exact error message (start LocalStack with `DEBUG=1` for debug logs), and please also share a full, self-contained example that helps us reproduce the issue. Thanks

**Comment by whummer** [comment]
@Tasemu Any updates on this? The `!FindInMap` function should be supported in Localstack - would be great if we could get a small, self-contained reproducible test case, as well as the error details you're seeing. Thanks.

**Comment by whummer** [comment]
@Tasemu This should be fixed in #2249 - can you please give it a try with the `latest` version of the Docker image. Please report here if the problem persists. Thanks
